<?php

namespace ContainerRbLB6pX;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class get_ServiceLocator_6RWgksLService extends App_KernelDevDebugContainer
{
    /**
     * Gets the private '.service_locator.6RWgksL' shared service.
     *
     * @return \Symfony\Component\DependencyInjection\ServiceLocator
     */
    public static function do($container, $lazyLoad = true)
    {
        return $container->privates['.service_locator.6RWgksL'] = new \Symfony\Component\DependencyInjection\Argument\ServiceLocator($container->getService, [
            'manager' => ['services', 'doctrine.orm.default_entity_manager', 'getDoctrine_Orm_DefaultEntityManagerService', false],
            'passwordHasher' => ['privates', 'security.user_password_hasher', 'getSecurity_UserPasswordHasherService', true],
            'user' => ['privates', '.errored..service_locator.6RWgksL.App\\Entity\\User', NULL, 'Cannot autowire service ".service_locator.6RWgksL": it references class "App\\Entity\\User" but no such service exists.'],
        ], [
            'manager' => '?',
            'passwordHasher' => '?',
            'user' => 'App\\Entity\\User',
        ]);
    }
}
